Library/Homebrew/os/mac/xcode.rb

```diff
@@ -282,6 +298,17 @@ def version
         end
       end
 
+      # Version string of the header package, which is a
+      # separate package as of macOS 10.14.
+      def headers_version
+        if !separate_header_package?
+          version
+        else
+          @header_version ||= MacOS.pkgutil_info(EXECUTABLE_PKG_ID)[/version: (.+)$/, 1]
+          @header_version.nil? ? ::Version::NULL : ::Version.new(@header_version)
```

Comment:
I'd make this an `if` block or early `return` as the ternary plus `::` makes this hard to mentally parse.